.github/workflows/beep-test.yml

```diff
@@ -0,0 +1,40 @@
+name: Testing - main
+
+on: [push, pull_request]
+
+jobs:
+  build:
```

Comment:
Definitely.  Will change the other one to linting.  I noticed that they were both the same thing when I was using act, so hopefully this can disambiguate if we need to do postprocessing of tests, etc.